README.md

```diff
@@ -10,6 +10,16 @@ https://samtecspg.github.io/articulate/
 
 ## QUICK START
 
+**If installing on a public host or on a docker host that uses a 192.168, etc type IP please ensure you create a .env file and put the below values in it so the application will work as expected**
+
+`DOCKER_HOST=ip_address_here
```

Comment:
This isn't formatted properly. code blocks should be triple back ticks. Either way I think we should just put a link to the docs for the cloud notes. The fewer docs we have to maintain in two places the better.